subprojects/build-cache/src/main/java/org/gradle/caching/configuration/BuildCache.java

```diff
@@ -34,12 +34,12 @@
     void setEnabled(boolean enabled);
 
     /**
-     * Returns whether pushing to the build cache is enabled.
+     * Returns whether storing in build cache is enabled.
      */
     boolean isPush();
 
     /**
-     * Sets whether pushing to the build cache is enabled.
+     * Sets whether storing in the build cache is enabled.
```

Comment:
Perhaps "Configure whether a given build can store outputs in the build cache". 

I think the important detail is that this is configured per-build.

Reply:
I would even go for "Whether a given build can store outputs in the build cache." WDYT?

Reply:
Here you go: https://github.com/gradle/gradle/pull/5411